deploy/cluster-role-bindings.yaml

```diff
@@ -3,11 +3,12 @@ apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRoleBinding
 metadata:
   name: pgo-cluster-role
+  namespace: "$PGO_OPERATOR_NAMESPACE"
 roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: ClusterRole
   name: pgo-cluster-role
 subjects:
 - apiGroup: rbac.authorization.k8s.io
   kind: User
-  name: system:serviceaccount:$PGO_OPERATOR_NAMESPACE:postgres-operator
+  name: system:serviceaccount:"$PGO_OPERATOR_NAMESPACE":postgres-operator
```

Comment:
Is this the right place for quotes? It seems like two `"` will be inside this value now.

Formatting would be simpler if we used `kind: ServiceAccount` which allows a separate field for namespace:
```yaml
- kind: ServiceAccount
  name: postgres-operator
  namespace: "$PGO_OPERATOR_NAMESPACE"
```